assigner/config/__init__.py

```diff
@@ -60,4 +60,8 @@ def __getattr__(self, key):
         if key == "roster" and self.data.get(key, None) is None:
             self.data[key] = []
 
+        # Fill in a blank course list if needed
+        elif key == "canvas-courses" and self.data.get(key, None) is None:
+            self.data[key] = []
+
```

Comment:
Let's remove this and let `canvas.py` and friends handle it. (I'd also like to eventually remove the roster check above but that's a different problem for a different day.)